tests/actions/fetchLabOrderAction.test.js

```diff
@@ -0,0 +1,66 @@
+import fetchLabOrders from '../../app/js/actions/fetchLabOrders';
+
+describe('fetch lab order action test-suite', () => {
+  beforeEach(() => moxios.install());
+  afterEach(() => moxios.uninstall());
+  const limit = 10;
+  const startIndex = 0;
+  const patient = '761d8414-6cb4-4456-be4f-232e6f819e06';
```

Comment:
A `constants.js` file was created for variables that could be easily changed, please take this off and add to that file.

Reply:
When writing tests, I try as much as possible not to use codes outside the `test environment`, I prefer to create a manual mock of every data required for my tests and the `constants.js` file is for use in the actual code, and not for tests.

using the `constants.js` file would also infer that I am not practicing `TDD` as I would be touching a file outside the test environment before writing tests.

Reply:
Cool

Reply:
+1 to what @Efosaok said... this is really just mock data that is meaningless outside of the test, not a global constant.